data/common.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 profile_metrics_alerting::alert_cycle::crons: {}
 profile_metrics_alerting::alert_cycle::enable_cycle_alerts: false
-profile_metrics_alerting::alert_cycle::script_path: "/root/grafana_tools/bulk_alert/alert_toggle.sh"
+profile_metrics_alerting::alert_cycle::script_path: "/root/grafana_tools/bulk_alert/unified_alerts/silence.sh"
```

Comment:
I think we also need to update the grafana_tools file under `files/root/grafana_tools/bulk_alert` in this repo. 

The `grafana_tools` files are included in this `profile-metrics_alerting` module repo. So, we want to update those files as part of this change too so that they include `bulk_alert/unified_alerts/silence.sh` and any other files.

Reply:
My above comment was listed as pending the last few days until I just submitted it a min ago... I don't remember having to do that when commenting before, must have added my comments in a different way than usual

Reply:
Updated the files in files/root/grafana_tools/bulk_alert to pull in updates from https://git.ncsa.illinois.edu/ici-monitoring/grafana-tools